ports/stm32/Justfile

```diff
@@ -131,6 +131,18 @@ flash version="1.0.0" screen="mono" ext="": (sign version screen ext) && (reset)
         SIGNED_FILENAME="v{{version}}-passport-fe-key-user.bin"
     fi
 
+    SIZE=$(ls -l build-Passport/$SIGNED_FILENAME | cut -d " " -f5)
+
+    # 1792k firmware size limit
+    LIMIT=1835008
+    echo "limit: $LIMIT"
+    echo "size: $SIZE"
+    if [ $SIZE -gt $LIMIT ]
+    then
+        echo "Firmware too large: $SIZE, limit: $LIMIT"
```

Comment:
I would put `ERROR: ` in front of the message.  We also don't really need to repeat the size or limit since they are immediately above this error.